### DISCUSSION TITLE
How to continuously render a component without input?

### DISCUSSION THREAD

**Discussion by OMANOMNOM** [ArthurSonzogni/FTXUI#646] [Q&A] [answered]
Before i start i should probably mention that i'm very new to C++, so excuse my ignorance. 

Basicly i have a component which is made up of some gauges, i was hoping to render them every 60th of a second, but instead they only update/render when i move the mouse. ScreenInteractive doesn't seem to offer me a way to manually render a component and Screen only seems to render elements, not components. 

``` 
if (selected == 0) {  
  start = std::chrono::high_resolution_clock::now();
  screen.ExitLoopClosure();  
  curScreen = &renderLoadingScreen;  
  screen.Loop(*curScreen);  
  return nullptr;  
	}
```

Is the a way to override this. I could make them elements and just manually render them, but i'd like to use screeninterative as the rest of my components use it. 

Cheers

**Comment by pralad-p** [comment, marked as answer]
This discussion #555 should answer your question.
